docs/statistics.md

```diff
@@ -111,11 +119,31 @@ The time elapsed, in milliseconds, since the SRT socket has been created (after
 
 ### pktSentTotal
 
-The total number of sent data packets, including retransmitted packets. Available for sender.
+The total number of sent DATA packets, including retransmitted packets ([pktRetransTotal](#pktRetransTotal)). Available for sender.
+
+If the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts sent packet filter control packets ([pktSndFilterExtraTotal](#pktSndFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
 
 ### pktRecvTotal
 
-The total number of received packets, including retransmitted packets. Available for receiver.
+The total number of received DATA packets, including retransmitted packets ([pktRcvRetransTotal](#pktRcvRetransTotal)). Available for receiver.
+
+If the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts received packet filter control packets ([pktRcvFilterExtraTotal](#pktRcvFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
+
+### pktSentUniqueTotal 
+
+The number of unique DATA packets sent by the SRT sender. Available for sender. 
+
+This value contains only unique **original** DATA packets. Retransmitted DATA packets ([pktRetransTotal](#pktRetransTotal)) are not taken into account. If the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](https://cac-word-edit.officeapps.live.com/we/API.md)), packet filter control packets ([pktSndFilterExtraTotal](#pktSndFilterExtraTotal)) are also not taken into account.
+
+This value corresponds to the number of original DATA packets sent by the SRT sender. It counts every packet sent over the network for the first time, and can be calculated as follows: `pktSentUniqueTotal = pktSentTotal – pktRetransTotal`, or by `pktSentUniqueTotal = pktSentTotal – pktRetransTotal - pktSndFilterExtraTotal` if the  `SRTO_PACKETFILTER` socket option is enabled. The original DATA packets are sent only once. 
+
+### pktRecvUniqueTotal 
+
+The number of unique original or retransmitted DATA packets **received in time** by the SRT receiver and, as a result, successfully delivered to the application. Available for receiver.
+
+This value contains only **unique** (meaning "first arrived") DATA packets. There is no difference whether a packet is original or retransmitted (in case of loss). Whichever packet comes first is taken into account. This statistic doesn't count duplicate packets (retransmitted or sent several times by defective hardware/software) or arrived too late packets (retransmitted or original packets arrived out of order) and, as a result, dropped by the TLPKTDROP mechanism (see [pktRcvDropTotal](#pktRcvDropTotal) statistic).
```

Comment:
```suggestion
This value contains only **unique** (meaning "first arrived") DATA packets. There is no difference whether a packet is original or retransmitted (in case of loss). Whichever packet comes first is taken into account. This statistic doesn't count duplicate packets (retransmitted or sent several times by defective hardware/software) or arrived too late packets (retransmitted or original packets arrived out of order) and, as a result, dropped by the TLPKTDROP mechanism or packets failed to be decrypted (see [pktRcvDropTotal](#pktRcvDropTotal) statistic).
```

Reply:
Yes, decrypt-failed packets are also not counted. This is consistent with the initial statement that this counts packets that are `successfully delivered to the application`. Undecrypted packets are dropped and never delivered.

Reply:
I've updated the description.